build/run-in-docker.sh

```diff
@@ -62,18 +62,25 @@ else
   PLATFORM_FLAG=
 fi
 
-env && hostname && uname -a
+echo "..printing env & other vars to stdout"
+echo "HOSTNAME=`hostname`"
+uname -a
+env
 echo "DIND_ENABLED=$DOCKER_IN_DOCKER_ENABLED"
-echo "FLAGS=$FLAGS"
-echo "PLATFORM=$PLATFORM"
-echo "PLATFORM_FLAG=$PLATFORM_FLAG"
+echo "done..printing env & other vars to stdout"
 
 if [[ "$DOCKER_IN_DOCKER_ENABLED" == "true" ]]; then
-  echo "..reached DIND check TRUE block, inside run-in-docker.sh. Checking environ"
-  go get github.com/onsi/ginkgo/ginkgo@v1.16.4
+  echo "..reached DIND check TRUE block, inside run-in-docker.sh"
+  echo "FLAGS=$FLAGS"
+  go env
+  set -x
+  go install -mod=mod github.com/onsi/ginkgo/ginkgo@v1.16.4 
+  find / -name ginkgo 2>/dev/null
```

Comment:
```suggestion
  find / -type f -name ginkgo  2>/dev/null
```

I think we should add file type?

Reply:
added `-type f` flag to find. Please check